pages/apply/[pid].js

```diff
@@ -44,6 +61,15 @@ export async function getServerSideProps({ params }) {
   };
 }
 
+export const getAdvertSchemeVersion = async (contentfulSlug) => {
+  return await axios
+    .get(
+      `${process.env.APPLICANT_BACKEND_HOST}/grant-adverts/${contentfulSlug}/scheme-version`,
+    )
+    .catch(function (error) {
+      return error;
+    });
```

Comment:
Dont think we need this catch if all we're doing is throwing the error

Reply:
This is returning rather than throwing the error so the consuming code can examine the status that's been returned